styles/controllers/extrasOnDemand.less

```diff
@@ -8,24 +8,86 @@
  * Copyright (c) 2014-2015 Simon Fraser University Library
  * Copyright (c) 2003-2015 John Willinsky
  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
- * 
+ *
  * @ingroup pkp_controllers_extrasOnDemand
  *
  * @brief Styles for pkp_controllers_extrasOnDemand
  */
 
 .pkp_controllers_extrasOnDemand {
-	.ui-icon {
-		float:left;
+	display: inline-block;
+
+	> a {
+		position: relative;
+		display: block;
+		padding: 0 @base 0 (@double + @double);
+		border: @bg-border;
+		line-height: @double + @base;
+		text-decoration: none;
+
+		.fa {
+			position: absolute;
+			top: 0;
+			left: 0;
+			line-height: @double + @base;
+			width: @double + @base;
+			text-align: center;
+			border-right: @bg-border;
+
+			&:before {
+				display: inline-block;
+				transition: transform 0.3s;
+			}
+		}
 	}
-	.toggleExtras {
-		margin-top:15px;
-		margin-bottom:15px;
-		cursor:pointer;
-		float:left;
-		/* The following allows us to
-		   implement a graceful fallback in case
-		   JS is disabled. */
+
+	> .container {
+		padding: 0 @base 0 (@double + @double);
+		max-height: 0;
+		transition: padding 0.3s;
+		overflow: hidden;
+	}
+
+	.toggleExtras-active {
```

Comment:
sorry, the class uses the active word but display is none. I thought it was a mistake, but your gif proves it's working :P Just curious about the logic.